api/src/main/java/org/openmrs/module/aijar/metadata/core/PublicHoliday.java

```diff
@@ -0,0 +1,56 @@
+package org.openmrs.module.aijar.metadata.core;
```

Comment:
@reagan-meant move this class to the org.openmrs.module.ugandaemr package since its not metadata

Reply:
@ssmusoke I cant seem to find the package you have mentioned...did you mean org.openmrs.module.aijar

Reply:
@reagan the package is there! you have put it in the metadata.core sub-package

Reply:
Done....But I think you have referenced a different person with the id *Reagan* mine is reagan-meant